app/controllers/meal_compensations_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
     @worktimes = Worktime.includes(:employee)
                          .in_period(@period)
                          .where(meal_compensation: true)
+                         .order('employees.firstname', 'employees.lastname')
```

Comment:
An anderen Orten sortieren wir nach lastname, dann firstname (z.B. [auf der Memberliste](time.puzzle.ch//employee_master_data)).. Wäre das hier eine Option?

```suggestion
                         .order('employees.lastname', 'employees.firstname')
```

Reply:
Wurde von Bruno in der Reihenfolge gewünscht.

Reply:
Alles klar :+1: